src/ophyd_async/core/detector.py

```diff
@@ -245,17 +241,21 @@ async def describe(self) -> Dict[str, DataKey]:
 
     @AsyncStatus.wrap
     async def trigger(self) -> None:
+        # set default trigger_info
+        self._trigger_info = TriggerInfo(
+            num=1, trigger=DetectorTrigger.internal, deadtime=0.0, livetime=0.0
+        )
```

Comment:
What happens if you have previously run `prepare(TriggerInfo(trigger=DetectorTrigger.internal, livetime=0.5))` as a way of setting up for a step scan with 0.5s exposure?